broker-core/src/main/java/io/zeebe/broker/logstreams/state/ZbColumnFamilies.java

```diff
@@ -0,0 +1,73 @@
+/*
+ * Zeebe Broker Core
+ * Copyright © 2017 camunda services GmbH (info@camunda.com)
+ *
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU Affero General Public License as published by
+ * the Free Software Foundation, either version 3 of the License, or
+ * (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU Affero General Public License for more details.
+ *
+ * You should have received a copy of the GNU Affero General Public License
+ * along with this program.  If not, see <http://www.gnu.org/licenses/>.
+ */
+package io.zeebe.broker.logstreams.state;
+
+public enum ZbColumnFamilies {
```

Comment:
I like in general that we use enums instead of strings to specify column families, I'm not a fan that we have this huge enum class used across all state classes when really, a state only cares about its specified sub-list of column families. I don't really have a better suggestion though at the moment...some way to compose enums out of other enums? :see_no_evil: